pymc3_hmm/utils.py

```diff
@@ -458,3 +457,59 @@ def plot_predictive_histograms(
             )
 
     return axes
+
+
+def plot_split_timeseries_hists(
+    plot_data: pd.DataFrame,
+    sample_col: str,
+    sample_mean_col: str,
+    twin_column_name: Union[str, None] = None,
+    observed_col: Union[str, None] = None,
```

Comment:
This `observed_col` is another thing that can&mdash;and should&mdash;be added by the user.